crates/walrus-service/src/client/daemon/routes.rs

```diff
@@ -78,6 +79,8 @@ pub const QUILT_PATCH_BY_ID_GET_ENDPOINT: &str = "/v1/blobs/by-quilt-patch-id/{q
 /// The path to get blob from quilt by quilt ID and identifier.
 pub const QUILT_PATCH_BY_IDENTIFIER_GET_ENDPOINT: &str =
     "/v1/blobs/by-quilt-id/{quilt_id}/{identifier}";
+/// The path to list patches in a quilt.
+pub const LIST_PATCHES_IN_QUILT_ENDPOINT: &str = "/v1/quilts/patches-by-id/{quilt_id}";
```

Comment:
Nit: From the path it's a bit unclear what ID is provided. Maybe `patches-by-quilt-id`?